applications/luci-app-ddns-go/htdocs/luci-static/resources/view/ddns-go.js

```diff
@@ -84,6 +84,11 @@ return view.extend({
 		o.default = '300';
 		o.rmempty = false;
 
+        o = s.option(form.Value, 'dns', _('DNS Server'), _('Specify the DNS server address to solve the error of the net/http: TLS handshake timeout.'));
+        o.datatype = 'ipaddr'; 
+        o.default = ''; 
+        o.rmempty = true; 
+
```

Comment:
```suggestion
		o = s.option(form.Value, 'dns', _('Custom DNS server'));
		o.datatype = 'ipaddr'; 
```

use tab for indentation, remove useless description, move it below `insecure`.